docs/test_plan.md

```diff
@@ -0,0 +1,59 @@
+# Test Plan
+Orangelight has good coverage by automated tests, but there may be some scenarios in
+which manual tests may also be necessary.  Below are the steps to test important
+workflows in this application manually.
+
+## Basic Search
+
+|Step|Action|Expected result|
+|---|---|---|
+|1|Go to [the application](https://catalog-staging.princeton.edu/)|The application has a Search box at the top of the page|
+|2|Enter `plasticity` in the search box and press the enter key or click the search button|You are taken to a search results page with a number of matching results|
+|3|Click "Catalog" in the header to return to the home page|
+|4|Select `Author (keyword)` from the dropdown in the Search bar|
+|5|Enter `john d clark` in the search box and press the enter key or click the search button|You are taken to a search results page with a number of matching results|
+
+## Record Page
+
+|Step|Action|Expected result|
+|---|---|---|
+|1|Go to the Record page for [an item](https://catalog-staging.princeton.edu/catalog/99111163333506421)|The application displays a page with details on the book It's all Greek|
+|2|There should be a section with author, format, language, etc. for the item|
+|3|There should be a section with the holding location, call number, availability information, and a request button|
+|4|Click the `Where to find it` button|A modal should pop up with a map of firestone with a marker indicating the shelving location of the item|
```

Comment:
can we add one more step between 4 and 5: click on the x button to close the modal. returns to the record page

Reply:
Done!